lib/logstash/outputs/loggly.rb

```diff
@@ -72,6 +82,13 @@ class LogStash::Outputs::Loggly < LogStash::Outputs::Base
   config :proxy_password, :validate => :password, :default => ""
 
 
+  # HTTP constants
+  http_success = "200"
```

Comment:
these are variables, not constants. In ruby, constants are indicated by capital letters: `HTTP_SUCCESS = "200"` which tells Ruby to not let oyu reassign their value.

Recommend capitalizing this and the following 4 lines of names